engine/src/main/java/org/terasology/rendering/dag/WireframeTask.java

```diff
@@ -16,29 +16,20 @@
 package org.terasology.rendering.dag;
```

Comment:
A note to @emanuele3d: Seems like there is a problem with this file, my experimentation got pushed. Let me take care of this one. Please don't consider this file.